website/docs/components/form/text-input/partials/code/how-to-use.md

```diff
@@ -118,9 +113,14 @@ It's possible to provide more than one error message using the more specific `Me
 </Hds::Form::TextInput::Field>
 ```
 
-##### Custom control ID
+#### Custom control ID
 
-In case it's necessary to have custom ID for the control, instead of the one automatically generated by the component (eg. because it needs to be referenced in the code for other reasons), you just need to pass a `@id` argument to the "field":
+In case it's necessary to have custom ID for the control, instead of the one automatically generated by the component (eg. because it needs to be referenced in the code for other reasons), you need to pass a `@id` argument to the field.
```

Comment:
```suggestion
If it’s necessary to have custom ID for the control, instead of the one automatically generated by the component (i.e., because it needs to be referenced in the code for other reasons), pass the `@id` argument to the field.
```